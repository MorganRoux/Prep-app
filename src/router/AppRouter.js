
import React from 'react'
import DashBoardPage from '../components/DashboardPage';
import LoginPage from '../components/LoginPage';
import {Router, Switch} from 'react-router-dom';
import PublicRoute from '../router/PublicRoute';
import PrivateRoute from '../router/PrivateRoute';
import createHistory from 'history/createBrowserHistory';

export const history = createHistory();

const AppRouter = () => (
    <Router history={history}>
    <div>
        <Switch>
            <PublicRoute path="/" component={LoginPage} exact={true} />
            <PrivateRoute path="/dashboard" component={DashBoardPage}/>
        </Switch>
        </div>
    </Router>
);

//export const AppRouter = () => (routes);

export default AppRouter;