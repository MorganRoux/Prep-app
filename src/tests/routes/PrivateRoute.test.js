import { PrivateRoute } from '../../router/PrivateRoute';
import React from 'react';
import { shallow } from 'enzyme';

test('should render privateRoute', () => {
    const wrapper = shallow (<PrivateRoute />);
    expect(wrapper).toMatchSnapshot();
})