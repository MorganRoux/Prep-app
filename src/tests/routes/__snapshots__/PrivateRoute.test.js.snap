// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render privateRoute 1`] = `
<Route
  component={[Function]}
/>
`;
